appnexus/cursor.py

```diff
@@ -39,21 +38,11 @@ def __getitem__(self, idx):
 
     def __iter__(self):
         """Iterate over all AppNexus objects matching the specifications"""
+        retrieved = 0
```

Comment:
Comment from @rambobinator 
The `retrieved` variable is not used anymore, remove it

Reply:
Variable removed in last commits